### PR TITLE
vox quills and a fancy bridge

### PR DESCRIPTION
**Pull request by improvedname**

### CODE REVIEW

File: code/modules/sprite_accessories/_accessory_hair.dm [hunk 1]
```diff
@@ -645,6 +645,58 @@
...
 
+/datum/sprite_accessory/hair/crestedquill
+	name = "Crested Quills"
+	icon_state = "crestedquills"
+
+/datum/sprite_accessory/hair/emperorquill
+	name = "Emperor Quills"
+	icon_state = "emperorquills"
+
+/datum/sprite_accessory/hair/keelquill
```
Comment (Trilbyspaceclone) [inline code/modules/sprite_accessories/_accessory_hair.dm]: You have this twice I think

Comment (improvedname) [reply code/modules/sprite_accessories/_accessory_hair.dm]: nope

Comment (improvedname) [reply code/modules/sprite_accessories/_accessory_hair.dm]: One is called Keel Other is called Keet

Comment (Trilbyspaceclone) [reply code/modules/sprite_accessories/_accessory_hair.dm]: My bad

**Pull request merged**